### PR TITLE
Home screen

### PR DESCRIPTION
**Pull request by RositaHamidi**
Please check your respective parts to make sure it is same as master branch as of this moment!

### CODE REVIEW

File: app/src/main/java/com/example/tryggaklassenpod/languages/contactus/ContactEng.kt [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(200.dp)
+            )
+
+            Spacer(modifier = Modifier.height(16.dp))
+
+            // Text content
+            Text(
+                text = "Are you interested in using the Tryggra Klassen at school? Send us a message, e-mail or call and we can tell you more about us, the concept, and the digital tools we use.",
```
Comment (Liz0001) [inline app/src/main/java/com/example/tryggaklassenpod/languages/contactus/ContactEng.kt:52]: all cards are duplicated code. Can be moved to one composable and the text sent in

Comment (RositaHamidi) [reply app/src/main/java/com/example/tryggaklassenpod/languages/contactus/ContactEng.kt:52]: I want to focus to making them all works for this weeks meeting and after that I am going to make them use one

File: app/src/main/java/com/example/tryggaklassenpod/languages/abouts/AboutSwe.kt [hunk 2]
```diff
@@ -1,2 +1,121 @@
...
+    Column (
+        modifier = Modifier.fillMaxWidth()
+            .verticalScroll(rememberScrollState()),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Spacer(modifier = Modifier.padding(10.dp))
+
+        Text(text = "Trygga Klassen Podcasten är din viktigaste källa för insikter som förbättrar " +
+                "välmåendet för barn och unga. Genom engagerande samtal med experter behandlar vi " +
+                "kritiska ämnen som missbruk, mobbning, och mental hälsa, samt ämnen som rör " +
```
Comment (Liz0001) [inline app/src/main/java/com/example/tryggaklassenpod/languages/abouts/AboutSwe.kt]: A lot of duplicate code with the language parts.
Would make more sense to send the necessary text from  res/strings to the reusable language component

Comment (RositaHamidi) [reply app/src/main/java/com/example/tryggaklassenpod/languages/abouts/AboutSwe.kt]: Another reason I wanted to merge before I continue so it won't affect the master!!! But apparently I have to deal with something that has nothing to do with me :|

Comment (RositaHamidi) [reply app/src/main/java/com/example/tryggaklassenpod/languages/abouts/AboutSwe.kt]: please check your code and let me know if there is anything affecting your part

**Pull request merged**